### PR TITLE
noImplicitAny: Sub 3000 errors

### PR DESCRIPTION
**Pull request by tskarhed**
#14714
Really getting into a flow here.

### CODE REVIEW

**Comment by dprokop** [comment]
<3

File: public/app/plugins/datasource/opentsdb/datasource.ts [hunk 1]
```diff
@@ -32,10 +39,10 @@ export default class OpenTsDatasource {
   }
 
   // Called once per panel (graph)
-  query(options) {
+  query(options: { rangeRaw: { from: any; to: any }; timezone: any; targets: any[] }) {
```
Comment (torkelo) [inline public/app/plugins/datasource/opentsdb/datasource.ts]: change options type to DataQueryRequest

**Review by torkelo — APPROVED** [review]
Looks great, just one small thing to improve in the opentsdb datasource

File: public/app/plugins/datasource/opentsdb/datasource.ts (hunk 1 above)
Comment (dehrax) [reply public/app/plugins/datasource/opentsdb/datasource.ts]: I added some changes to make it work!

**Pull request merged**